Proposals/xgov-31.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Add arbitrary decimal precision exp, log, pow to the AVM
+author: 1m1 (1m1-github)
+company_name: 2i2i
+category: AVM
+focus_area: Defi, Science, Math
+open_source: Yes
+amount_requested: 200000
+status: Draft
```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.